sql-parser/src/main/java/io/crate/sql/parser/Statement.g

```diff
@@ -1280,6 +1300,7 @@ KEY: 'KEY';
 DUPLICATE: 'DUPLICATE';
 SET: 'SET';
 RESET: 'RESET';
+DEFAULT: 'DEFAULT';
```

Comment:
it's already in this list https://crate.io/docs/reference/sql/reference/lexical_structure.html
should I add it somewhere else ?